packages/backend/src/core/AccountService.ts

```diff
@@ -1,20 +1,33 @@
 import { EthereumAddress } from '@explorer/types'
 
+import { ForcedTradeOfferRepository } from '../peripherals/database/ForcedTradeOfferRepository'
 import { PositionRepository } from '../peripherals/database/PositionRepository'
 
 export class AccountService {
-  constructor(private readonly positionRepository: PositionRepository) {}
+  constructor(
+    private readonly positionRepository: PositionRepository,
+    private readonly offerRepository: ForcedTradeOfferRepository
+  ) {}
 
   async getAccount(address?: EthereumAddress) {
     if (!address) {
       return undefined
     }
     const id = await this.positionRepository.findIdByEthereumAddress(address)
-    const hasUpdates = false // TODO: find forced actions in progress
+
+    if (!id) {
+      return {
+        address,
+      }
+    }
+
+    const pendingCount = await this.offerRepository.countPendingByPositionIdA(
+      id
+    )
     return {
       address,
       positionId: id,
-      hasUpdates,
+      hasUpdates: !!pendingCount,
```

Comment:
```suggestion
      hasUpdates: pendingCount > 0,
```

That's more straightforward